buildSrc/src/main/java/edu/wpi/gripgenerator/FileParser.java

```diff
@@ -216,43 +224,79 @@ public Expression getDefaultValue(String defaultValue) {
+        )
+                .setDirectionDefaults(DefinedParamType.DefinedParamDirection.INPUT_AND_OUTPUT, "dst")
+                .setIgnoreDefaults("dtype");
+
+        DefinedMethodCollection collection2 = new DefinedMethodCollection(baseClassName,
+                new DefinedMethod("scaleAdd", false, "Mat", "double", "Mat", "Mat")
+                        .addDescription("Calculates the weighted sum of two arrays."),
+                new DefinedMethod("normalize", false, "Mat", "Mat")
+                        .addDescription("Normalizes the norm or value range of a Mat"),
+                new DefinedMethod("addWeighted", false, "Mat")
+                        .addDescription("Calculates the weighted sum of two arrays."),
                 new DefinedMethod("flip", false,
                         new DefinedParamType("Mat"),
                         new DefinedParamType("Mat"),
-                        new DefinedParamType("int").setLiteralDefaultValue("Y_AXIS")),
-                new DefinedMethod("bitwise_and", false, "Mat", "Mat"),
-                new DefinedMethod("bitwise_or", false, "Mat", "Mat"),
-                new DefinedMethod("bitwise_xor", false, "Mat", "Mat"),
-                new DefinedMethod("bitwise_not", false, "Mat", "Mat"),
-                new DefinedMethod("absdiff", false, "Mat", "Mat"),
-                // TODO: Fix (Causes Segfault)
-                //new DefinedMethod("inRange", false),
+                        new DefinedParamType("int").setLiteralDefaultValue("Y_AXIS"))
+                        .addDescription("Flips a 2D array around vertical, horizontal, or both axes."),
+                new DefinedMethod("absdiff", false, "Mat", "Mat")
+                        .addDescription("Calculates the per-element absolute difference between two arrays."),
                 new DefinedMethod("compare", true,
                         new DefinedParamType("Mat"),
                         new DefinedParamType("Mat"),
                         new DefinedParamType("Mat"),
                         new DefinedParamType("int").setLiteralDefaultValue("CMP_EQ")
-                ),
-                new DefinedMethod("max", false, "Mat", "Mat"),
+                ).addDescription("Performs the per-element comparison of two Mats"),
+                new DefinedMethod("max", false, "Mat", "Mat")
+                        .addDescription("Calculates per-element maximum of two Mats"),
                 new DefinedMethod("min", false, "Mat", "Mat")
+                        .addDescription("Calculates per-element minimum of two Mats"),
+                new DefinedMethod("mulTransposed", false, "Mat", "Mat")
+                        .addDescription("Calculates the product of a matrix and its transposition. " +
+                                "The optional delta matrix subtracted from src before the " +
+                                "multiplication."),
+                new DefinedMethod("transpose", false, "Mat", "Mat")
```

Comment:
These non-image mat operations are not useful right now because we have nothing that deals with non-image mats